### ISSUE TITLE
AudioBook section/library in Ampache - Multible artists for same songs

### ISSUE THREAD

**Issue by szemlicka** [ampache/ampache#1996]
Hi,

I know that this request exists long time ago and required a complete re-design of UI,SQL,queries and so on.

But I also know that Ampache 4.0 will come. Is there a chance to get that function in 4.0?

I will explain why its a nice feature for some library.

I will use it for my AudioBooks. With Audiobooks I have also some series. Like Perry Rhodan Siberedition or Andreas Brandhorst with the Kantaki Saga. To find that in my LMS (Logitech Media Server), I tag it in that way:

Track: Diamant (Die Kantaki-Saga 1) (001)
Artist: Andreas Brandhorst
Album: Diamant (Die Kantaki-Saga 1)
Album-Artist: Andreas Brandhorst;Die Kantaki-Saga
CD: 1
Album Sort:0001

With that tagging, you can search for the artist Die Kantaki-Saga and you will find every album for that series. And you also know that the original artist is Andreas Brandhorst.
All albums are sorted based on Album Sort.

LMS can handle that very good. Booksonic also handle that more or less good.

Since some years, audiobooks get more and more popular. I think its a good idea to implement function for this kind of files too. They have also different requirements. So it makes no sense to implement shuffle for this files.

To implement a special AudioBook section in Ampache can make the different to all other solutions

**Comment by lachlan-00** [comment]
I agree that audiobooks are different enough that we should add a separate catalog type for them.

This would at least need new UI, based on probably podcasts and i would say new tables to handle a different structure as they are not the same as a music album and usually have a lot of chapters.

**Comment by kuzi-moto** [comment]
What about multiple artists for regular music? Do you think that could ever happen or is it going to take too much effort?

**Comment by lachlan-00** [comment]
There will be a way around it.

One thing that will make it easier is MBID's are a set length so having those available will probably make it easier to split. The content of the fields for artists are treated "as is" so it doesn't do any further inspection on the data other than the complete string.

All i think we would need to work on is grouping through splitting those up.

**Comment by szemlicka** [comment]
Why you don't handle this with the artist not in the same way like genre? A separator like ; can do the work. You only need to update the GUI. Or? Lady Gaga; Bradley Cooper - Shallow (A Star Is Born) is a good example for same song and 2 artists.

**Comment by lachlan-00** [comment]
MBID is a better way to do things as the codes don't change based on the case/order of the artists

Bradley Cooper = cbe7746f-d4b4-4f0d-a435-1aa181d97b1b
Lady Gaga = 650e7db6-b795-4eb5-a702-5ea2fc46c848

You could have multiple titles for the same artist groups but the mbid's will always be the same
```
Lady Gaga feat Bradley Cooper = 650e7db6-b795-4eb5-a702-5ea2fc46c848;cbe7746f-d4b4-4f0d-a435-1aa181d97b1b
Bradley Cooper and Lady Gaga = cbe7746f-d4b4-4f0d-a435-1aa181d97b1b;650e7db6-b795-4eb5-a702-5ea2fc46c848
```

MBID's are also the same length so if they aren't separated correctly they can be split safely.

**Comment by szemlicka** [comment]
I take a look to MBID and a lot of audio book artists are not inside MBID. So I think for audio books not a given solution...:-(

**Comment by lachlan-00** [comment]
Audio books need to be separated from music.
Multi artists would be for regular songs

**Comment by lachlan-00** [comment]
closing to start to reduce the amount of open enhancement issues.

I do want to eventually be able to make these changes but will focus on bigs and issues before releasing 4.0.

Keep an eye on #1986 in the mean time and maybe someone else can pick it up.

**Comment by Jieiku** [comment]
One thing about this, both audio books and podcasts use chapters.

Would be great if Ampache supported Audio books, and would also be great if chapters worked from the podcasts.
